internal/serve/httphandler/payment_handler.go

```diff
@@ -0,0 +1,89 @@
+package httphandler
+
+import (
+	"net/http"
+
+	"github.com/stellar/go/support/render/httpjson"
+	"github.com/stellar/wallet-backend/internal/data"
+	"github.com/stellar/wallet-backend/internal/entities"
+	"github.com/stellar/wallet-backend/internal/serve/httperror"
+	"github.com/stellar/wallet-backend/internal/services"
+)
+
+type PaymentsHandler struct {
+	Models  *data.Models
+	Service *services.PaymentService
```

Comment:
A pattern that we are for handlers is that we always use an interface to the services it uses, so it's easy to mock and test here.

Reply:
nit: can we rename the `PaymentsHandler` to `PaymentHandler`?